packages/lib-classifier/src/store/DrawingStore.js

```diff
@@ -1,56 +1,156 @@
-import { addDisposer, getRoot, onAction, types } from 'mobx-state-tree'
+import cuid from 'cuid'
+import { addDisposer, getRoot, getType, onAction, types } from 'mobx-state-tree'
 import { autorun } from 'mobx'
 import { Subject } from 'rxjs'
 
+import { Line, Point } from './markings'
+
+const markModels = {
+  line: Line,
+  point: Point
+}
+
+function getMarkModel (toolType) {
+  return markModels[toolType] || null
+}
+
 const DrawingStore = types
   .model('DrawingStore', {
```

Comment:
I'm not 100% following why there's a `DrawingStore` model and a `DrawingTask` model. Does this model the drawing itself ie. the marks created by all drawing tasks from the current workflow?